sharktank/sharktank/models/llama/tools/import_quark_dataset.py

```diff
@@ -0,0 +1,366 @@
+# Copyright 2024 Advanced Micro Devices, Inc
```

Comment:
Would be great to get some unit tests for this

Reply:
hmm

I could generate a safetensors file with one attn block of random numbers,  upstream a config json, and then save to irpa and load and compare against the initial random values. I'm not sure how useful that is as a test though. Do you have something else in mind?